src/Toggle/GetAllToggleStatusForUser.php

```diff
@@ -0,0 +1,71 @@
+<?php
+namespace Clearbooks\LabsApi\Toggle;
+
+use Clearbooks\Dilex\JwtGuard\IdentityProvider;
+use Clearbooks\Dilex\Endpoint;
+use Clearbooks\Labs\Toggle\Object\GetAllToggleStatusRequest;
+use Clearbooks\Labs\Toggle\UseCase\GetAllToggleStatus;
+use Clearbooks\LabsApi\User\Group;
+use Clearbooks\LabsApi\User\RawSegmentDataToSegmentObjectConverter;
+use Clearbooks\LabsApi\User\User;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+class GetAllToggleStatusForUser implements Endpoint
+{
+
+    /**
+     * @var GetAllToggleStatus
+     */
+    private $getAllToggleStatus;
+
+    /**
+     * @var IdentityProvider
+     */
+    private $identityProvider;
+
+    /**
+     * @var RawSegmentDataToSegmentObjectConverter
+     */
+    private $rawSegmentDataToSegmentObjectConverter;
+
+    /**
+     * @param GetAllToggleStatus $getAllToggleStatus
+     * @param IdentityProvider $identityProvider
+     * @param RawSegmentDataToSegmentObjectConverter $rawSegmentDataToSegmentObjectConverter
+     */
+    public function __construct( GetAllToggleStatus $getAllToggleStatus, IdentityProvider $identityProvider,
+                                 RawSegmentDataToSegmentObjectConverter $rawSegmentDataToSegmentObjectConverter )
+    {
+        $this->getAllToggleStatus = $getAllToggleStatus;
+        $this->identityProvider = $identityProvider;
+        $this->rawSegmentDataToSegmentObjectConverter = $rawSegmentDataToSegmentObjectConverter;
+    }
+
+    /**
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function execute( Request $request )
+    {
+        $userId = $this->identityProvider->getUserId();
+        if ( !isset( $userId ) ) {
```

Comment:
It's not used in every request, in an upcoming PR I don't use a check like this at all. Plus in some already existing requests the check and the text are different.